book-crawler/app/crawler/kyobo.py

```
@@ -110,7 +150,7 @@ def crawl_book_detail(driver: webdriver.Chrome, url: str) -> dict | None:
         try:
             review_tab = driver.find_element(By.XPATH, "//a[contains(@class, 'tab_link') and contains(., '리뷰')]")
             driver.execute_script("arguments[0].click();", review_tab)
-            time.sleep(2)
+            human_delay(base=2.0, spread=1.0)
         except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)